python/composio/tools/base/runtime.py

```diff
@@ -273,6 +273,12 @@ def _build_executable_from_args(  # pylint: disable=too-many-statements
 ) -> t.Tuple[t.Callable, t.Type[BaseModel], t.Type[BaseModel], bool]:
     """Build execute action from function arguments."""
     argspec = inspect.getfullargspec(f)
+    missing_annot = set(argspec.args) - set(argspec.annotations)
```

Comment:
Consider extracting this validation logic into a separate helper function for better readability and reusability, e.g.:
```python
def _validate_type_annotations(argspec: inspect.FullArgSpec) -> None:
    missing_annot = set(argspec.args) - set(argspec.annotations)
    if missing_annot:
        raise InvalidRuntimeAction(
            message=f"Type annotations are required for all parameters. Missing annotations for: {missing_annot}"
        )
```